iris-mpc-upgrade-hawk/bin/iris_mpc_hawk_genesis.rs

```diff
@@ -149,10 +153,36 @@ fn parse_args() -> Result<ExecutionArgs> {
         true
     };
 
+    // Arg: use_backup_as_source (parse as string, convert to bool for ExecutionArgs).
+    let use_backup_as_source = if let Some(use_backup_as_source_arg) =
```

Comment:
For consistency, I'd rather use the same argument parsing logic as the other booleans (e.g. `peform_snapshot` above